app/src/main/res/layout/activity_register.xml

```diff
@@ -1,217 +1,220 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:orientation="vertical"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
+              xmlns:app="http://schemas.android.com/apk/res-auto"
+              android:layout_width="match_parent"
+              android:layout_height="match_parent"
+              android:background="@drawable/table_bg"
 
-    android:background="@drawable/table_bg">
+              android:orientation="vertical">
 
     <TextView
         android:layout_width="170dp"
         android:layout_height="40dp"
         android:layout_marginLeft="20dp"
         android:padding="5dp"
         android:text="Register here"
-        android:textSize="20dp"
-        android:textColor="#e0f7fa"/>
+        android:textColor="#e0f7fa"
+        android:textSize="20dp"/>
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="450dp"
         android:layout_marginLeft="30dp"
-        android:layout_marginTop="10dp"
         android:layout_marginRight="30dp"
+        android:layout_marginTop="10dp"
         android:background="@drawable/login_bg"
         android:orientation="vertical"
         android:weightSum="6"
         >
 
         <com.rengwuxian.materialedittext.MaterialEditText
+            android:id="@+id/et_name"
             android:layout_width="match_parent"
             android:layout_height="0dp"
-            android:layout_weight="1"
-            android:hint="name"
+            android:layout_marginRight="20dp"
```

Comment:
MarginRight and MarginLeft is deprecated now

Reply:
@therealshabi Did the required changes.